WebApi/Controllers/OfficesController.cs

```diff
@@ -15,12 +15,22 @@ public class OfficesController : BaseApiController
     {        
 
         [HttpGet]
-        public async Task<IActionResult> GetAll()
+        public async Task<IActionResult> GetAll(bool? hasParking, string? query)
         {
-            var result = await Mediator.Send(new GetOfficeListQueryRequest());
+            var request = new GetOfficeListQueryRequest();
```

Comment:
you can put [FromQuery] Request request in method params

Reply:
Done

Reply:
Done